payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayRepository.kt

```diff
@@ -90,7 +95,11 @@ internal class DefaultGooglePayRepository(
                 ).toString()
             )
         }.getOrElse {
-            // TODO (samer-stripe): Add unexpected error event here
+            errorReporter.report(
```

Comment:
Is it possible to write a test for this one?

Reply:
Possibly. I could have `GooglePayJsonFactory` as a constructor dependency and mock it with a bad result. Unless the expected format from `IsReadyToPayRequest.fromJson`. WDYT?

Reply:
Your call.